tests/security/check_kernel_config/CC_STACKPROTECTOR_STRONG.pm

```diff
@@ -27,7 +27,16 @@ use utils;
 
 sub run {
     # check the kernel configuration file to make sure the parameter is there
-    validate_script_output "cat /boot/config-`uname -r`|grep CONFIG_STACKPROTECTOR", qr/CONFIG_STACKPROTECTOR_STRONG=y/;
+    if (!check_var('ARCH', 's390x')) {
+        validate_script_output "cat /boot/config-`uname -r`|grep CONFIG_STACKPROTECTOR", qr/CONFIG_STACKPROTECTOR_STRONG=y/;
+    }
+    else {
+        my $results = script_run("grep CONFIG_STACKPROTECTOR_STRONG=y /boot/config-`uname -r`");
+        if (!$results) {
+            die("Error: the kernel paramter is wrong configured on s390x platform");
```

Comment:
typo: s/paramter/parameter/, Adjective => Adverb s/wrong/wrongly/
Error: the kernel parameter is wrongly configured on s390x platform

Reply:
Thanks, my typo